src/cards/ha-weather-card.js

```diff
@@ -134,7 +165,9 @@ class HaWeatherCard extends LocalizeMixin(EventsMixin(PolymerElement)) {
               >
                 <div>
                   [[localize('ui.card.weather.attributes.air_pressure')]]:
-                  [[stateObj.attributes.pressure]] [[getUnit('air_pressure')]]
+                  <div class="unit">
```

Comment:
`ltr` does change things when you are in RTL mode because `rtl` is applied on the entire document and so it distorts some elements. putting `ltr` on it forces it to stay readable.
About the div - no since we need an element that will align properly.

Reply:
I think I didn't understand your comment. Yes in LTR nothing has changed. Here's a shot:
<img width="391" alt="capture" src="https://user-images.githubusercontent.com/37745463/50847450-466d0b00-137a-11e9-944e-7d51eaccb459.PNG">

Reply:
But you are setting it to `display: inline-block` that would take care of the alignment right? And make it work with a `span`? I have no knowledge about rtl, so not sure...

Reply:
You were right. Span initially didn't work but then I styled it and forgot to go back. Fixed.